### PR TITLE
net: wifi_shell: Updated TWT flow id range

### PR DESCRIPTION
**Pull request by ajayparida**
TWT flow id is of 3 bit value, so range limited to 0 to 7.

### CODE REVIEW

**Comment by ajayparida** [comment]
@sachinthegreem

**Comment by ajayparida** [comment]
@sachinthegreen @rado17 @krish2718

**Review by rlubos — DISMISSED** [review]

File: include/zephyr/net/wifi_mgmt.h [hunk 1]
```diff
@@ -229,7 +229,7 @@ struct wifi_twt_params {
 };
 
 /* Flow ID is only 3 bits */
-#define WIFI_MAX_TWT_FLOWS 8
+#define WIFI_MAX_TWT_FLOWS 7
```
Comment (leewkb4567) [inline include/zephyr/net/wifi_mgmt.h:232]: Changing this definition will affect the array size of the following structure.

`struct wifi_ps_config {`
`	struct wifi_twt_flow_info twt_flows[WIFI_MAX_TWT_FLOWS];`

Comment (ajayparida) [reply include/zephyr/net/wifi_mgmt.h]: Yes, u[pdated the value and releated changes.

**Review by leewkb4567 — COMMENTED** [review]
Work fine with the nRF7002 DK.

**Review by krish2718 — APPROVED** [review]

**Review by rado17 — APPROVED** [review]

**Review by rlubos — APPROVED** [review]

**Review by jukkar — APPROVED** [review]

**Pull request merged**